tests/unit/amo/components/TestAddon.js

```diff
@@ -1123,6 +1123,9 @@ describe(__filename, () => {
         { ...fakeAddon, slug: 'addon-1', id: 1 },
         { ...fakeAddon, slug: 'addon-2', id: 2 },
         { ...fakeAddon, slug: 'addon-3', id: 3 },
+        { ...fakeAddon, slug: 'addon-4', id: 4 },
+        { ...fakeAddon, slug: 'addon-5', id: 5 },
+        { ...fakeAddon, slug: 'addon-6', id: 6 },
```

Comment:
Would it be worthwhile to make this contain more than 6 add-ons, to test that the limit of 6 is actually being enforced?

Reply:
Oh, good call. I'll do that. 👍

Reply:
Actually, the add-ons by authors card does that bit. Arguably these aren't needed but I thought made the intentional clearer so I left them. 🤷‍♂️